infra/utils/validator/fake-run-and-compare.js

```diff
@@ -0,0 +1,115 @@
+const vm = require('vm');
```

Comment:
Kill this as well.

Reply:
See my previous comment about wanting to maintain testing parity in this PR.